src/adapter/src/coord/sequencer/linked_cluster.rs

```diff
@@ -12,10 +12,8 @@
 use std::time::Duration;
 
 use mz_compute_client::controller::ComputeReplicaConfig;
-use mz_controller::clusters::{
-    ClusterId, ReplicaAllocation, ReplicaConfig, ReplicaLogging,
-    DEFAULT_REPLICA_LOGGING_INTERVAL_MICROS,
-};
+use mz_controller::clusters::{ReplicaAllocation, ReplicaConfig, ReplicaLogging};
+use mz_controller_types::{ClusterId, DEFAULT_REPLICA_LOGGING_INTERVAL_MICROS};
```

Comment:
`DEFAULT_REPLICA_LOGGING_INTERVAL_MICROS` TODO

Reply:
This const is itself a `pub use` of `mz_compute_client::controller::DEFAULT_COMPUTE_REPLICA_LOGGING_INTERVAL_MICROS` (moved to mz_compute_types in this PR), which seemed intentional, so I left it alone.

Happy to e.g. remove the re-export and inline it if you think that's better?

Reply:
It is weird that the controller defines a constant that's only used by the `adapter` and `sql` crates. Seems like it should be defined in one of those instead?